spring-cloud-alibaba-sentinel/pom.xml

```diff
@@ -42,6 +42,12 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.cloud</groupId>
+            <artifactId>spring-cloud-commons</artifactId>
+            <optional>true</optional>
+        </dependency>
+
```

Comment:
Maybe the scope of this dependency should be provided and using @ConditionalOnClass to annotate SentinelCircuitBreakerConfiguration.